src/use-connect.js

```diff
@@ -4,21 +4,21 @@ import useScript from 'react-script-hook';
 const MAX_POLL_NUMBER = 3;
 const POLLING_INTERVAL = 100;
 
-const src = 'https://utility-connect-main.prod.arcadia.com/dist/v3.js';
+const src = 'https://connect-main.prod.arcadia.com/dist/v3.js';
```

Comment:
Need to make double check this URL and version after `Connect` bundle changes are made 👍

Reply:
Should be all set!